install/kubernetes/helm/istio/charts/security/values.yaml

```diff
@@ -46,3 +46,11 @@ enableNamespacesByDefault: true
 # "security" and value "S1".
 podAntiAffinityLabelSelector: []
 podAntiAffinityTermLabelSelector: []
+
+# This parameter sets the CLI parameter append-dns-names of
+# Citadel, which:
+# - If true, Citadel will generate webhook certificates.
+# - If true, Citadel will not generate webhook certificates.
```

Comment:
should be `- If false, Citadel will not generate webhook certificates.` in the comment.